data/ch/contact.yml

```diff
@@ -0,0 +1,29 @@
+############################## Contact #############################
+contact:
+  enable : true
+  title : Contactez-nous
+  subtitle : Détails du contact
+  content : Nous aimons parler des problèmes complexes de mise à l'échelle et de performances. Qu'est-ce qui vous ralentit? Écrivez-nous pour une discussion amicale. Nous en ferons la peine.
+
+  # Contact details
+  # Themify icon pack : https://themify.me/themify-icons
+  contact_list:
+    - icon : ti-email
+      info : "info@rotational.io"
+    - icon : ti-twitter
+      info : "@rotationalio"
+    - icon : ti-github
+      info : "rotationalio"
+    - icon : ti-linkedin
+      info : rotational
+
+  # # Contact form
+  # # Works with https://formspree.io
+  # contact_form:
+  #   enable : true
+  #   form_action : "https://formspree.io/f/xknpawwn"
+  #   name : "Votre nom"
+  #   email : "Votre email"
+  #   subject : "Matière"
+  #   message : "Message"
+  #   submit : "Soumettre"
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)